flambe/nn/cnn.py

```diff
@@ -122,18 +122,44 @@ def __init__(self,
                     raise ValueError("Kernel size tuple should have same length as conv_dim")
 
             layer = conv_block(
-                conv(prev_c, c, k, stride, padding),
+                conv(prev_c, c, k, stride, padding),  # type: ignore
                 activation,
-                pooling,
                 dropout,
+                pooling,
```

Comment:
Pooling after dropout will alter the dropout effect... at least in my intuition. Is there a reason to do this?

Reply:
Yes, i don't want the last returned vector to have dropout, I want the dropout to remain somewhat internal. So if someone wants to add another dropout between the result of the encoder and the final projection (say for a classifier), they can